### ISSUE TITLE
How can I change the activeOpacity of the picker?

### ISSUE THREAD

**Issue by wnikola** [hossein-zare/react-native-dropdown-picker#423]
It seems that when pressed the picker behaves like a TouchableOpacity with activeOpacity set way too high for my liking, is it possible to change this?

**Comment by hossein-zare** [comment]
Hello,

In a future release that will be added.

**Comment by hossein-zare** [comment]
`props` and `itemProps` are available in **v5.2.2**.
```jsx
<DropDownPicker
  ...
  props={{
    activeOpacity: 0.5
  }}
  itemProps={{
    activeOpacity: 0.5
  }}
/>
```

**Comment by wnikola** [comment]
Thanks!
